packages/core/src/endowmentsToolkit.js

```diff
@@ -114,6 +121,14 @@ function endowmentsToolkit({
           }
           allowedWriteFields.add(path)
           whitelistedReads.push(path)
+          if (packagePolicyValue === 'write+define') {
+            if (!unwrapFrom) {
+              throw Error(
```

Comment:
(doesn't really matter)

```suggestion
              throw ReferenceError(
```